src/client/components/UnreadMessages/Unread.style.css

```diff
@@ -0,0 +1,16 @@
+.number-of-unread-messages {
+  border: solid burlywood 1px;
+  width: 20px;
+  height: 20px;
+  margin: auto;
+  border-radius: 50%;
+  background-color: burlywood;
+  position: relative;
+}
+
+.font-weight {
```

Comment:
Two comments:
- Use scoping: `.number-of-unread-messages p`
- Please don't name class `font-weight`, it is not descriptive of what it is styling.